### DISCUSSION TITLE
Standalone or offline mode?

### DISCUSSION THREAD

**Discussion by pgifford** [DannyDainton/newman-reporter-htmlextra#378] [Q&A] [answered]
Our development environment is not connected to the Internet and never will be.  The HTML output loads resources (js and css files) from CDN sites which of course fails.  We have the resources available on our intranet; is there a way to configure the reporter to load from alternate locations or a hardcoded path?

**Comment by DannyDainton** [comment, marked as answer]
Without those js scripts and CSS' hardcoded into the template, it's not going to work. 

You _could_ create a new custom template and add those directly to the file, it's obviously going to be huge as it's bringing a lot of different things.

**Reply by pgifford** [reply]
Thanks!  I should have thought of that; we use Handlebars on our project 🤦‍♂️  I appreciate the prompt reply!
